NetKVM/Common/ParaNdis-TX.h

```diff
@@ -105,6 +107,31 @@ class CNB : public CNdisAllocatableViaHelper<CNB>
     DECLARE_CNDISLIST_ENTRY(CNB);
 };
 
+typedef struct _tChainOfNbls
+{
+    CNdisRefCounter m_NumChained;
+    CNdisRefCounter m_NumCompleted;
+    CNBL *m_FirstInChain;
+} CChainOfNbls;
+
+class NBLHistory : public CNdisAllocatable<NBLHistory, 'NBLH'>
+{
+    DECLARE_CNDISLIST_ENTRY(NBLHistory);
+  public:
+    NBLHistory(LPCSTR Func, LPCSTR Title, LPCSTR ParameterMeaning, PVOID Parameter, LPCSTR ValueMeaning, ULONG Value);
+  protected:
```

Comment:
same here

Reply:
@Jedoku   is this behavior configurable? I ask because this does not make too much sense.

Reply:
It's configured in .clang-format 
EmptyLineBeforeAccessModifier: LogicalBlock
If we change it it will affect whole project

Reply:
@Jedoku If there is an option "don't mind" this will not request any change, correct? Is it?

Reply:
`Leave` option should work